<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Bart Cerneels</author>
 <class>PodcastSettingsBase</class>
 <widget class="QWidget" name="PodcastSettingsBase">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>305</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Podcast Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="m_urlLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>URL:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_saveLocationlabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Save Location:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="m_urlLineEdit"/>
       </item>
       <item>
        <widget class="KUrlRequester" name="m_saveLocation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="m_autoFetchCheck">
     <property name="toolTip">
      <string>When checked, Amarok will automatically scan the podcast for updates</string>
     </property>
     <property name="whatsThis">
      <string>When checked, Amarok will automatically scan the podcast for updates</string>
     </property>
     <property name="text">
      <string>Automatically scan for updates</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KButtonGroup" name="kbuttongroup">
     <property name="title">
      <string>Media Download</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="m_downloadRadio">
        <property name="toolTip">
         <string>Download media as soon as it becomes available</string>
        </property>
        <property name="whatsThis">
         <string>Download media as soon as it becomes available</string>
        </property>
        <property name="text">
         <string>Download when a&amp;vailable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_streamRadio">
        <property name="toolTip">
         <string>Media must be explicitly downloaded, otherwise the podcast will be played from the remote server.</string>
        </property>
        <property name="whatsThis">
         <string>Media must be explicitly downloaded, otherwise the podcast will be played from the remote server.</string>
        </property>
        <property name="text">
         <string>Stream or download on re&amp;quest</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="m_purgeCheck">
       <property name="toolTip">
        <string>If checked, Amarok will throw away old podcast episodes</string>
       </property>
       <property name="whatsThis">
        <string>If checked, Amarok will throw away old podcast episodes</string>
       </property>
       <property name="text">
        <string>Limit &amp;number of episodes</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer16">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>108</width>
         <height>29</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_purgeCountLabel">
       <property name="text">
        <string>Keep maximum of:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KIntSpinBox" name="m_purgeCountSpinBox">
       <property name="toolTip">
        <string>The maximum number of podcast items to store</string>
       </property>
       <property name="whatsThis">
        <string>The maximum number of podcast items to store</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m_writeTagsLayout">
     <item>
      <widget class="QCheckBox" name="m_writeTagsCheck">
       <property name="whatsThis">
        <string>The RSS feed and the tags in the downloaded file sometimes do not contain the same information. Writing the tags ensures the information in the playlist and on media devices will be the same as in the feed.</string>
       </property>
       <property name="text">
        <string>Write feed information to tags after downloading.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="m_filenameLayoutConfigWidgetButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Episode Filename Configuration</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">kurlrequester.h</include>
  <include location="local">klineedit.h</include>
  <include location="local">kpushbutton.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>m_purgeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_purgeCountSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_purgeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_purgeCountLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
